client/homescreen/activity-panel/orders/index.js

```diff
@@ -155,20 +160,33 @@ class OrdersPanel extends Component {
 
 		const cards = [];
 		orders.forEach( ( order ) => {
-			const extendedInfo = order.extended_info || {};
+			const {
+				date_created_gmt: dateCreatedGmt,
+				extended_info: extendedInfo,
+				order_id: orderId,
+				total_sales: totalSales,
+			} = order;
 			const productsCount =
 				extendedInfo && extendedInfo.products
 					? extendedInfo.products.length
 					: 0;
 
-			const total = order.total_sales;
+			const total = totalSales;
 
 			cards.push(
 				<ActivityCard
-					key={ order.order_id }
+					key={ orderId }
 					className="woocommerce-order-activity-card"
 					title={ orderCardTitle( order ) }
-					date={ order.date_created_gmt }
+					date={ dateCreatedGmt }
+					onClick={ ( element ) => {
```

Comment:
Its a tiny thing but this should always be named `event` to avoid confusion because it's not an element. It can also be destructured here if you like.

```suggestion
					onClick={ ( event ) => {
```

or with destructuring:

```suggestion
					onClick={ ( { target } ) => {
```

Reply:
Nice catch! This was changed in the commit 226756e